engine/classes/Elgg/Security/Crypto.php

```diff
@@ -42,7 +42,7 @@ class Crypto {
 	 */
 	public function getRandomString($length, $chars = null) {
 		if ($length < 1) {
-			throw new ElggInvalidArgumentException('Length should be >= 1');
+			throw new RangeException('Length should be >= 1');
```

Comment:
Length exception?

Reply:
No length exceptions apply to the length of the param